src/services/comments.js

```diff
@@ -32,6 +27,13 @@ module.exports = class ProjectsHelper {
 				})
 			}
 
+			//update the comment
+			if (bodyData.status === common.STATUS_RESOLVED || !bodyData.resolved_by) {
+				bodyData.resolved_by = loggedInUserId
+				bodyData.resolved_at = new Date()
+				bodyData.status = common.STATUS_RESOLVED
```

Comment:
It is already checking the status right? do we have to assign it again? @priyanka-TL

Reply:
its optional they can pass either status or resolved_by